src/main/java/org/mskcc/limsrest/util/Utils.java

```diff
@@ -490,6 +507,10 @@ private static String getMostAdvancedSampleStatus(DataRecord sample, String requ
                 int currentStatusOrder = getSampleTypeOrder(currentSampleType.toLowerCase());
                 long currentRecordId = current.getRecordId();
                 if (isSequencingComplete(current, user)) {
+                    // Check if Sample has failed Sequencing analysis
+                    if (hasFailedSequencingDataQc(current, user)){
+                        return String.format("%s%s", WORKFLOW_STATUS_FAILED, STAGE_SEQUENCING_ANALYSIS);
```

Comment:
`org.mskcc.limsrest.util.StatusTrackerConfig.STAGE_SEQUENCING_ANALYSIS`

Reply:
Added the change requested.